tests/core/test_opds2_import.py

```diff
@@ -283,18 +287,53 @@ def test_opds2_importer_correctly_imports_valid_opds2_feed(
             == huckleberry_finn_delivery_mechanisms[1].delivery_mechanism.content_type
         )
 
+        # 2.3 Edition with non open-access acquisition links (The Politics of Postmodernism)
+        postmodernism_license_pool = self._get_license_pool_by_identifier(
+            pools, self.PROQUEST_IDENTIFIER
+        )
+        assert True == isinstance(postmodernism_license_pool, LicensePool)
+        assert False == huckleberry_finn_license_pool.open_access
```

Comment:
Did you mean `postmodernism_license_pool.open_access` here?

Reply:
👍🏻 fixed